.github/workflows/ossf_scorecard.yml

```diff
@@ -0,0 +1,48 @@
+name: Scorecard supply-chain security
+on:
+  # For Branch-Protection check. Only the default branch is supported. See
+  # https://github.com/ossf/scorecard/blob/main/docs/checks.md#branch-protection
+  branch_protection_rule:
+  schedule:
+    - cron: "0 9 * * *"
+  push:
+    branches: [develop]
```

Comment:
We don't have this branch - probably should be `main`

Reply:
sssshhh 😂